core/templates/pages/exploration-editor-page/services/user-email-preferences.service.spec.ts

```diff
@@ -36,81 +34,79 @@ describe('User Email Preferences Service', function() {
     }
   };
 
-  beforeEach(function() {
-    angular.mock.module('oppia');
+  var serviceInstance: UserEmailPreferencesService = null;
+  var httpTestingController: HttpTestingController = null;
+  var csrfService: CsrfTokenService = null;
```

Comment:
I think generally, the naming is done where the variable name is the same as the service, with just the first letter as small.  Seems to have been incorrect in the original code itself, can we fix that?

Reply:
Can you please, elaborate more? Do you want to change serviceInstance to userEmailPreferencesService or make instance's name and service's name same?

Reply:
Fixed.